src/si/momentum.rs

```diff
@@ -0,0 +1,202 @@
+//! Momentum (base unit kilogram meters per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+
+quantity! {
+    /// Momentum (base unit kilogram meters per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+    quantity: Momentum; "momentum";
+    /// Momentum dimension, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+    dimension: ISQ<
+        P1,     // length
+        P1,     // mass
+        N1,     // time
+        Z0,     // electric current
+        Z0,     // thermodynamic temperature
+        Z0,     // amount of substance
+        Z0>;    // luminous intensity
+    units {
+        @yottagram_meter_per_second: prefix!(yotta) / prefix!(kilo); "Yg*m/s", "yottagram meter per second",
```

Comment:
Use  `·`  instead of `*`. Put a space on each side of the operator. No space around the `/`. (https://www.nist.gov/pml/special-publication-811/nist-guide-si-chapter-6-rules-and-style-conventions-printing-and-using)

Reply:
done